### DISCUSSION TITLE
First use report

### DISCUSSION THREAD

**Discussion by ferdiworks** [rrirower/DP2SceneGraphWizard#1] [General]
First thank you for the oppotunity and tool.
Second I tried the tool w a few different feeds that are active for my DP Channels 

Sofrtware download was good!
Setup was good
All worked good
output was good

I tried my first feed with shortform video and tagged categories
I tried a seconf feed with tvSpecials and playlist

Both failed to parse or display items on screen when side loaded!

Fingers crossed and looking at code!

Thanks again!

**Comment by rrirower** [comment]
Thanks for the feedback. If at all possible, can you provide the urls of the feeds that fail so I can test?  If you like, you can send it directly to me at rrirower@charter.net.

**Reply by ferdiworks** [reply]
Done a new test w tvSpecials and it worked, everythingf parsed. I may have unzippeed and rezipped the folder with first test and caused an issue! Now I want to trest the output file on sideload!

**Reply by ferdiworks** [reply]
Done another channel test with another tvSpecials feed and worked well - Both channels are active on Roku as DP

I want to tryt another with short form videos!

**Reply by ferdiworks** [reply]
OK Testing going well here. Issue found for me was my feed for tvspecials and short form videos that have liveFeed don't parse, when I remove liveFeed from the feeds the feeds parse fine!

**Reply by rrirower** [reply]
Again, if you can provide me with a URL of the feed that contains the liveFeed, I can step through the code and see what the issue may be,  I tested with a liveFeed that was not active and it seemed to work correctly.

**Reply by ferdiworks** [reply]
I found the issue!
Components - task - MainLoaderTask.brs - line 84
change - if ObjectContainsTags(movie) = true
change to = if ObjectContainsTags(liveFeed) = true

All works now!

**Reply by rrirower** [reply]
Thanks.  I've made that change in the code.

**Comment by samadtehrani** [comment]
Nice Job, I tried it with a simple DP feed. It got categories correctly but I had no luck with setting up tags. I will try it later this weekend. 
I'm working on the same concept too. I did fix Deeplinking and RAF but kind of hard-coded the series categories' names onto the manifest file and used the same names in my feed so by comparing these two I did set up my GridScreen. I will definitely look into your  MainLoadr.brs file and learn some stuff.
